docs/source/contribute/dev-assisted-package-creation.rst

```diff
@@ -0,0 +1,113 @@
+
+The data processing you've implemented cannot already exist in
+GeoIPS, or any of its corresponding packages. If you have processing routines that are
+similar, it is likely you'll just have to reformat your inputs to integrate correctly
+with the GeoIPS system. We are happy to help with that if needed. For a full list of
+public repositories that NRLMMD-GEOIPS supports, see
+`GeoIPS Repositories <https://github.com/orgs/NRLMMD-GEOIPS/repositories>`_.
+
+Please either submit an issue `here <https://github.com/NRLMMD-GEOIPS/geoips/issues/new/choose>`_,
+or contact us at ``geoips@nrlmry.navy.mil`` specifying your use case and the assistance
+needed. Here's what we'll need from you for either method.
+
+Information Needed for Developers
+---------------------------------
+
+In order to create a standalone GeoIPS Package implementing your routines, we'll need
+the following information:
+
+#. ``Data``
+
+   a. We'll take on most datasets that are needed to produce your expected output. The
+      more data and types of data which you provide, the better chance we have at
+      testing and producing accurate outputs that match your current routines. Please
+      don't provide us with Terabytes of data, only that which is necessary to produce
+      your product in its entirety.
+
+#. ``Code``
+
+   a. The code provided should encapsulate everything you or your group created to get
+      your processing routines working. This does not include code from dependent 3rd
+      party software packages, but a list of software dependencies would be appreciated
+      so we know what to install. Code included could be any of the following bullet
+      points.
+   b. Algorithms
+   c. Colormaps
+   d. Readers
+   e. Utility Modules
+   f. Any other code you wrote that's required to produce the expected output
+
+#. ``List of Input Variables`` needed to produce the desired output
+
+   a. Since the data you're using is geospatial, we'll need some context of what
+      variables we'll be dealing with. Please provide us with a list of variables needed
+      to produce your expected output. It should look something like what is shown below.
+   b. [``cloud_height_acha``, ``relative_humidity``, ``cloud_water_content``, ...]
+
+#. ``Examples of Accurate Outputs`` from your current routines.
+
+   a. Included alongside these outputs, please provide the data used to generate these
+      outputs. This way, we have a concrete test example to compare alongside to make
+      sure what we've implemented matches what you expect. Could Include:
+   b. Imagery
+   c. GeoTIFFs
+   d. NetCDF / HDF5 Files
+   e. Any other output that could assist with testing
+
+#. ``Description of your Geospatial Processing Routines``
+
+   a. In this description, you should include the following information:
+   b. What this software achieves
+   c. What outputs it creates
+   d. Why it's needed
+
+Once we have the aforementioned information, we'll be able to move forward in creating
```

Comment:
Maybe "we'll be able to more forward with helping you create".

Reply:
? I think one of these words might be a typo